docs/user/dashboard/lens.asciidoc

```diff
@@ -319,7 +319,12 @@ You can also change the index pattern for a single *Layer*.
 [[why-my-field-x-is-missing-from-the-fields-list]]
 ===== Why is my field X missing from the fields list?
 
-*Lens* does not support the visualization of full-text fields, therefore it is not showing them in the data summary.
+If your field is not in the `Available fields` list in *Lens* there may be multiple reasons, here described:
+* the field is a full-text field, therefore it is not showing in the data summary.
+* the field is a Geo field, therefore it is not showing in the data summary prior to v.7.14.0
```

Comment:
It is best practice to avoid calling out versions in the docs because it is hard to maintain. If there are version-specific docs, we only push PRs to those specific versions.

Reply:
This section is about why fields do not appear in the *Available fields* list, so I'm not sure we need to include information about data summaries.

Reply:
Is `Geo field` correct? I'm not familiar with Maps, but is this the terminology they use?

Reply:
`geo_point` field is the naming in the app:

<img width="549" alt="Screenshot 2021-07-14 at 14 43 39" src="https://user-images.githubusercontent.com/924948/125623982-5c884c3c-85e9-43ed-bed4-ac232821ffe3.png">

Reply:
> This section is about why fields do not appear in the _Available fields_ list, so I'm not sure we need to include information about data summaries.

You are right. Prior to 7.14 is wasn't showing up in the `Available fields`, not data summaries